src/main/java/htsjdk/variant/variantcontext/VariantContext.java

```diff
@@ -1663,8 +1663,11 @@ public String getContig() {
     }
 
     /**
-     * Returns 1-based inclusive start position of the variant, 0 or greater.
-     * See below for explanation on "0".
+     * Returns 1-based inclusive start position of the variant.
+     *
+     * <p>
+     *     INDEL events usually start on the first unaltered reference base before the INDEL.
```

Comment:
This is a bit confussing... INDEL event actual start in the bae before the INDEL... not really ... the indel start where it starts... perhaps you mean to say that it is reported one base before.
Also perhaps you should try to be more general here... instead of talking about INDEL alone you could say something like:
"Notice that for some types of variant events the actual start position may not be this value (e.g. deletions are reported on the base before the first base deleted)."

So you are not giving the impression that it may only happen with deletions or indels.

Reply:
This is directly copied from the old comment.

But I agree that it can be more general.